src/store/trees/audio.js

```diff
@@ -73,7 +73,18 @@ export default {
             commit('setRecordingStatusFor', {key, uuid, queueId, value: 'initiating_call'})
             setTimeout(() => dispatch('fetchAudioRecordingStatusFor', {key, uuid, queueId, isFirstCall: true}), 3000)
           })
-      // .catch(({response: {data: {status_description: message}}}) => )
+      .catch((error) => {
+        if (error.response) {
+          // Request made and server responded
+          console.log('Error Log', error.response.data.status_description.message);
+        } else if (error.request) {
+          // The request was made but no response was received
+          console.log('Error Log', error.request);
```

Comment:
Hi hi @safydy, for error logs, pls use like 
```console.error('audio', 'The request was made but no response was received', error)```

Reply:
sure, changing it. Thanks